.eslintrc.json

```diff
@@ -30,6 +34,8 @@
         ],
         "no-console": "off",
         "no-useless-escape": "off",
-        "no-var": "error"
+        "no-var": "error",
+        "no-new": "off",
+        "import/no-duplicates": "off"
```

Comment:
Why this?

Reply:
This is not required anymore since we have separated SVG files with proper spacing.